### DISCUSSION TITLE
How do I set Routing policy when creating a Route53 record

### DISCUSSION THREAD

**Discussion by balamuru** [aws/aws-sdk-java#2963] [Q&A] [answered]
When I use the Java Route53 API `ResourceRecordSet` to create a record, the default routing policy seems to be "Latency" . I haven't found an obvious way in the [API docs](https://docs.aws.amazon.com/fr_fr/AWSJavaSDK/latest/javadoc/com/amazonaws/services/route53/AmazonRoute53.html#changeResourceRecordSets-com.amazonaws.services.route53.model.ChangeResourceRecordSetsRequest-)  to set this policy to "Simple", for instance , . like I would be able to do on the AWS web UI - see screenshot for what I want to affect .

![rec](https://user-images.githubusercontent.com/73320/232882808-01c5ebf5-f615-497b-b16f-28f87d193fb4.png)

**Comment by bhoradc** [comment, marked as answer]
Hello @balamuru,
 
Using the first sample from the [documentation](https://docs.aws.amazon.com/fr_fr/AWSJavaSDK/latest/javadoc/com/amazonaws/services/route53/AmazonRoute53.html#changeResourceRecordSets-com.amazonaws.services.route53.model.ChangeResourceRecordSetsRequest-), you should be able to create the routing record with Simple Routing policy. 

Also, you can refer to the [Dev Guide](https://docs.aws.amazon.com/Route53/latest/DeveloperGuide/routing-policy.html) on how to choose the routing policy.

<img width="1345" alt="image" src="https://github.com/aws/aws-sdk-java/assets/122290643/5b0994d2-405c-4d52-b603-67d5adad475b">

Kindly follow the above highlighted sample and do let me know if you have further queries on it.

Regards,
Chaitanya
